app/components/home/Signin.js

```diff
@@ -32,18 +35,20 @@ export class Signin extends Component {
 
     if(!this.state.licenseId) {
```

Comment:
FYI - 'license_id' isn't a required field in the model, but I'm guessing it should be!